app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -6,47 +6,42 @@ class MonthlyKeyMetricsReport < BaseReport
 
     attr_reader :report_date
 
+    Report = Struct.new(:metadata, :table, :csv_name)
```

Comment:
just noodling on ideas for this
- `EmailableTable`
- `TabularReportingData`
and for `metadata` what if we called if `email_format_options`? since it's not quite generic metadata, it's specific to our `ReportMailer.tables_report`